src/test/scala/com/sksamuel/scapegoat/inspections/matching/PartialFunctionInsteadOfMatchTest.scala

```diff
@@ -83,10 +83,6 @@ class PartialFunctionInsteadOfMatchTest extends InspectionTest {
                      future onComplete {
                        case _ =>
                      }
-                     future onSuccess {
```

Comment:
This was throwing an error as well because `onSuccess` doesn't exist anymore in Scala 2.13.